cmd/deploy/deploy.go

```diff
@@ -511,9 +511,34 @@ func getDefaultTimeout() time.Duration {
 	return parsed
 }
 
-func GetDeployer(ctx context.Context, manifest *model.Manifest, opts *Options, builder *buildv2.OktetoBuilder, cmapHandler configMapHandler) (deployerInterface, error) {
+func shouldRunInRemote(opts *Options) bool {
+	// already in remote so we need to deploy locally
```

Comment:
@andreafalzetti `OKTETO_DEPLOY_REMOTE` is a const that we set to `true` [when we build our dockerfile deployer/destroyer](https://github.com/okteto/okteto/blob/master/cmd/deploy/remote.go#L59) to know that we are in a remote instance which means that we need to deploy locally because we are already in the remote. Probably name is a little bit confusing. Do you think we could rename it to `OKTETO_IS_DEPLOY_OR_DESTROY_ALREADY_IN_REMOTE` or split the const in two depending on the operation ([deploy](https://github.com/okteto/okteto/blob/master/cmd/deploy/remote.go#L59) vs [destroy](https://github.com/okteto/okteto/blob/master/cmd/destroy/remote.go#L58))? Maybe `OKTETO_IS_DEPLOY_ALREADY_IN_REMOTE` and `OKTETO_IS_DESTROY_ALREADY_IN_REMOTE`? WDYT?